mulighetsrommet-api/src/test/kotlin/no/nav/mulighetsrommet/api/repositories/TiltakstypeRepositoryTest.kt

```diff
@@ -224,4 +227,70 @@ class TiltakstypeRepositoryTest : FunSpec({
             totalCount shouldBe 105
         }
     }
+
+    context("Strukturert innhold for deltakerregistrering") {
+        val tiltakstyper = TiltakstypeRepository(database.db)
+
+        beforeEach {
+            tiltakstyper.upsert(TiltakstypeFixtures.Oppfolging)
+            tiltakstyper.upsert(TiltakstypeFixtures.VTA)
+            tiltakstyper.upsert(TiltakstypeFixtures.Arbeidstrening)
+        }
+
+        test("Skal hente ut korrekt strukturert innhold for tiltakstype som har strukturert innhold") {
+            @Language("PostgreSQL")
+            val query = """
+                insert into deltaker_registrering_innholdselement(innholdskode, tekst)
+                values('jobbsoking', '${TiltakstypeFixtures.Oppfolging.tiltakskode}')
+                on conflict do nothing;
+
+                insert into deltaker_registrering_innholdselement(innholdskode, tekst)
+                values('kartlegge-helse', '${TiltakstypeFixtures.Oppfolging.tiltakskode}')
+                on conflict do nothing;
+
+                update tiltakstype
+                set deltaker_registrering_ledetekst = 'Oppfølging er et bra tiltak'
+                where tiltakskode = '${TiltakstypeFixtures.Oppfolging.tiltakskode}';
+
+                insert into tiltakstype_deltaker_registrering_innholdselement(innholdskode, tiltakskode)
+                values('jobbsoking', '${TiltakstypeFixtures.Oppfolging.tiltakskode}');
+
+                insert into tiltakstype_deltaker_registrering_innholdselement(innholdskode, tiltakskode)
+                values('kartlegge-helse', '${TiltakstypeFixtures.Oppfolging.tiltakskode}');
+            """.trimIndent()
+            queryOf(
+                query,
+            ).asExecute.let { database.db.run(it) }
```

Comment:
Ser ut som at de to andre testene nå er avhengig av at denne koden har kjørt. Kan flyttes til `beforeEach` for å gjøre det tydeligere at dette er felles for alle tester.